doc/policy/packages.md

```diff
@@ -18,7 +18,7 @@ tip or some preceding transaction in the package.
 
 The following rules are enforced for all packages:
 
-* Packages cannot exceed `MAX_PACKAGE_COUNT=25` count and `MAX_PACKAGE_SIZE=101KvB` total size
+* Packages cannot exceed `MAX_PACKAGE_COUNT=25` count and `MAX_PACKAGE_SIZE=101KB` total size
```

Comment:
Would it be simpler / easier to understand if we just said there's a maximum weight `MAX_PACKAGE_WEIGHT=404000`?

Reply:
did MAX_PACKAGE_KWEIGHT=404